ledger/sync/src/network_state/polling_network_state.rs

```diff
@@ -7,53 +7,49 @@
 use crate::{NetworkState, SCPNetworkState};
 use mc_common::{
     logger::{log, Logger},
-    ResponderId,
+    NodeID,
 };
 use mc_connection::{
-    BlockchainConnection, Connection, ConnectionManager, RetryableBlockchainConnection,
-};
-use mc_consensus_scp::{
-    core_types::Ballot, msg::ExternalizePayload, Msg, QuorumSet, SlotIndex, Topic,
+    BlockchainConnection, Connection, ConnectionManager, RetryableBlockchainConnection, }; use mc_consensus_scp::{
+    core_types::Ballot, msg::ExternalizePayload, test_utils, Msg, QuorumSet, SlotIndex, Topic,
 };
 use mc_transaction_core::BlockIndex;
 use mc_util_uri::ConnectionUri;
 use retry::delay::{jitter, Fibonacci};
 use std::{
     collections::{HashMap, HashSet},
-    str::FromStr,
     sync::{Arc, Condvar, Mutex},
     thread,
     time::Duration,
 };
 
-// Since PollingNetworkState is not a full-fledged consensus node, it does not
-// have a local node id. However, quorum tests inside the scp crate require a
-// local node id, so we provide one. Ideally this should not be a node id that
-// can be used on a real network.
-const FAKE_NODE_ID: &str = "fake:7777";
-
 pub struct PollingNetworkState<BC: BlockchainConnection> {
     /// Connection manager (for consensus nodes we are going to poll).
     manager: ConnectionManager<BC>,
 
     /// SCPNetworkState instance that provides the actual blocking/quorum set
     /// check logic.
-    scp_network_state: SCPNetworkState<ResponderId>,
+    scp_network_state: SCPNetworkState<NodeID>,
 
     /// Logger.
     logger: Logger,
 }
 
 impl<BC: BlockchainConnection + 'static> PollingNetworkState<BC> {
     pub fn new(
-        quorum_set: QuorumSet<ResponderId>,
+        quorum_set: QuorumSet<NodeID>,
```

Comment:
Optional/Nit: Let's remove the type override -- `NodeID` is the default
```suggestion
        quorum_set: QuorumSet,
```